### ISSUE TITLE
Random "Failed to spawn IntelliSense process: 65520" on Mac with 1.13.3

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#10091]
Has anyone else seen this? We're investigating....

**Comment by H-G-Hristov** [comment]
I haven't seen this error bug it feels related to my previous complaint: https://github.com/microsoft/vscode-cpptools/issues/10090
Should it be a popup or a line in the log?

**Comment by sean-mcmanus** [comment]
@H-G-Hristov The error would appear in the C/C++ logging.

**Comment by sean-mcmanus** [comment]
@H-G-Hristov Also, this is the first I've heard about 1.13.3 being worse than 1.13.2. The wordexp issue is definitely fixed with 1.13.2. We don't know yet what the severity of this 65520 error is though.

**Comment by H-G-Hristov** [comment]
Well I couldn't find: Failed to spawn IntelliSense in th log and it looks like IntelliSense works but the flame is stuck there currently the hover displays: Running clang-tidy 1/2 (50%). 
![Screenshot 2022-11-02 at 0 12 34](https://user-images.githubusercontent.com/47526411/199351884-58c84628-641a-45c8-95ee-d146e14062b1.png)

**Comment by sean-mcmanus** [comment]
@H-G-Hristov Do you have the C_Cpp.codeAnalsysis.clangTidy.enabled and C_Cpp.codeAnalysis.runAutomatically both set to "true" or you are manually running it?

**Comment by H-G-Hristov** [comment]
> @H-G-Hristov Do you have the C_Cpp.codeAnalsysis.clangTidy.enabled and C_Cpp.codeAnalysis.runAutomatically both set to "true" or you are manually running it?

Yes. 
Yes.I think runAutomatically is on by default.

```json5
{
  "C_Cpp.codeAnalysis.clangTidy.enabled": true,
  "C_Cpp.loggingLevel": "Debug",
  "editor.rulers": [
    // A ruler with the default or editorRuler.foreground color at column 800
    80,
    {
      "column": 100,
      "color": "#546266d7"
    },
    {
      "column": 120,
      "color": "#ff000033"
    },
  ],
  "files.associations": {
    ".clang-format": "yaml",
    ".clang-tidy": "yaml",
    "*.cmake.in": "cmake",
    "array": "cpp",
    "tuple": "cpp",
    "utility": "cpp",
    "variant": "cpp",
    "cstdint": "cpp",
    "__bit_reference": "cpp",
    "__functional_base": "cpp",
    "__node_handle": "cpp",
    "algorithm": "cpp",
    "atomic": "cpp",
    "bitset": "cpp",
    "chrono": "cpp",
    "cstddef": "cpp",
    "deque": "cpp",
    "__memory": "cpp",
    "filesystem": "cpp",
    "functional": "cpp",
    "iterator": "cpp",
    "limits": "cpp",
    "locale": "cpp",
    "memory": "cpp",
    "optional": "cpp",
    "ratio": "cpp",
    "regex": "cpp",
    "system_error": "cpp",
    "type_traits": "cpp",
    "vector": "cpp",
    "__config": "cpp",
    "__debug": "cpp",
    "__errc": "cpp",
    "__hash_table": "cpp",
    "__locale": "cpp",
    "__mutex_base": "cpp",
    "__nullptr": "cpp",
    "__split_buffer": "cpp",
    "__string": "cpp",
    "__threading_support": "cpp",
    "__tree": "cpp",
    "__tuple": "cpp",
    "any": "cpp",
    "bit": "cpp",
    "cctype": "cpp",
    "cmath": "cpp",
    "codecvt": "cpp",
    "complex": "cpp",
    "concepts": "cpp",
    "condition_variable": "cpp",
    "csignal": "cpp",
    "cstdarg": "cpp",
    "cstdio": "cpp",
    "cstdlib": "cpp",
    "cstring": "cpp",
    "ctime": "cpp",
    "cwchar": "cpp",
    "cwctype": "cpp",
    "exception": "cpp",
    "coroutine": "cpp",
    "fstream": "cpp",
    "future": "cpp",
    "initializer_list": "cpp",
    "iomanip": "cpp",
    "ios": "cpp",
    "iosfwd": "cpp",
    "iostream": "cpp",
    "istream": "cpp",
    "list": "cpp",
    "map": "cpp",
    "mutex": "cpp",
    "new": "cpp",
    "ostream": "cpp",
    "queue": "cpp",
    "set": "cpp",
    "sstream": "cpp",
    "stack": "cpp",
    "stdexcept": "cpp",
    "streambuf": "cpp",
    "string": "cpp",
    "string_view": "cpp",
    "strstream": "cpp",
    "thread": "cpp",
    "typeindex": "cpp",
    "typeinfo": "cpp",
    "unordered_map": "cpp",
    "*.ipp": "cpp",
    "span": "cpp",
    "pointers": "cpp",
    "__bits": "cpp",
    "clocale": "cpp",
    "compare": "cpp",
    "numbers": "cpp"
  },
  "files.autoSave": "afterDelay",
  "testMate.cpp.test.advancedExecutables": [
    {
      "pattern": "{build,Build,BUILD,out,Out,OUT}/vscode.*/**/*{test,Test,TEST}*"
    }
  ],
  "githubPullRequests.ignoredPullRequestBranches": [
    "develop"
  ]
}
```

**Comment by H-G-Hristov** [comment]
> @H-G-Hristov Is the clang-tidy process actually running? If you run code analysis on a particular file, does it queue and finish or is it stuck? It might be a bug with the status completion reporting (i.e. it might incorrectly think there's more work to do when there isn't).

Yesterday I checked activity monitor but I did't see a clang-tidy process.
I need to make some observations first to answer the questions.

What I am experiencing right now. After I start VSCode and I open a C++ file. Clang-tidy runs and displays some results. I continue working in the code editor - random stuff - editing, opining more files, and after a while I don't see the yellow squiggles anymore and the flame is stuck forever. I didn't observe such behavior in 1.13.2 I think. Before that I had a bunch of other issues which you fixed and I believe that 1.13.2 behaved overall rather reasonably.

**Comment by sean-mcmanus** [comment]
This 65520 issue happens very rarely and probably reproed with 1.13.2 as well but we didn't notice due to the lower repro probability than the other issues we hitting and fixed. It's most likely unrelated to your issue, so we'll continue to track that code analysis issue with https://github.com/microsoft/vscode-cpptools/issues/10090 .
